src/Tinkoff.Trading.OpenApi/Tinkoff.Trading.OpenApi.csproj

```
@@ -12,6 +12,7 @@
         <RepositoryType>git</RepositoryType>
         <PackageTags>Tinkoff invest trading</PackageTags>
         <Description>SDK for Tinkoff Invest</Description>
+        <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
```

сейчас это делается по-другому
```suggestion
        <IncludeSymbols>true</IncludeSymbols>
        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```
https://docs.microsoft.com/ru-ru/nuget/create-packages/symbol-packages-snupkg